src/arch/mod.rs

```diff
@@ -10,14 +10,16 @@ use cranelift::codegen::{
     },
     isa::{CallConv, TargetIsa},
 };
-use lazy_static::lazy_static;
 use target_lexicon::Triple;
 
+use lazy_static::lazy_static;
+pub(crate) use x64::*;
+use Type::*;
+
```

Comment:
Please don't reorder these imports.

Reply:
Forgot to remove "Reorganize Imports" in CLion. It sorts them by scope though, don't know what's bad about it?

Reply:
I guess it's not a big deal but it distracts from the actual changes. We can keep these ones.